labgrid/driver/power/poe_mib.py

```diff
@@ -0,0 +1,27 @@
+""" tested with Cisco CBS350, should be compatible with switches implementing the PoE administration MiB"""
+
+from ..exception import ExecutionError
+from ...util.snmp import SimpleSNMP
+
+OID = "1.3.6.1.2.1.105.1.1.1.3.1"
+
+def power_set(host, port, index, value):
+    _snmp = SimpleSNMP(host, 'private')
+    outlet_control_oid = "{}.{}".format(OID, index)
```

Comment:
Could be an f-string.

Reply:
Yeah, do you want it to be one? I don't care either way tbh.

Reply:
Let's leave it as-is.